content/docs/other-guides/multi-user-overview.md

```diff
@@ -17,12 +17,66 @@ Note that the isolation support in Kubeflow doesn't provide any hard security
 guarantees against malicious attempts by users to infiltrate other user's
 profiles.
 
+## Design overview
+
+Kubeflow multi-tenancy implementation currently follows:
```

Comment:
The markdown is not correctly rendering as a list.
https://deploy-preview-1603--competent-brattain-de2d6d.netlify.com/docs/other-guides/multi-user-overview/